AUTHORS.rst

```diff
@@ -4,8 +4,8 @@ Credits
 
 This is a joint LSST-DESC and LSST-SLSC pipeline project.
 
-Contributors
-------------
+Contributors (alphabetical)
+---------------------------
 
 * Simon Birrer `sibirrer <https://github.com/sibirrer/>`_
-
+* Aprajita Verma `sibirrer <https://github.com/aprajita/>`_
```

Comment:
sorry - another one 1 - sibirrer after my name to aprajita

Reply:
uff, good catch! That's what reviews are for :) thank you very much, just updated it